obspy/core/inventory/response.py

```diff
@@ -735,10 +736,227 @@ def __init__(self, resource_id=None, instrument_sensitivity=None,
             msg = "response_stages must be an iterable."
             raise ValueError(msg)
 
-    def get_evalresp_response_for_frequencies(
-            self, frequencies, output="VEL", start_stage=None, end_stage=None):
+    @classmethod
+    def from_resp(cls, resp, frequency=None, sr=None):
```

Comment:
Sure, I understand that logic, keep imports pointing uphill. 
I was thinking if you want a response, you should use the Response class and a factory for Response objects should live in response.